setup.py

```diff
@@ -128,6 +128,9 @@ def _format_pulp_requirement(plugin, specifier=None, ref=None, gh_namespace="pul
     "marshmallow<4.0.0,>=3.6.1",
     "django-picklefield<4.0.0,>=3.0.1",
     "django-ipware<4.0.0,>=3.0.0",
+    # to fix a compatibility issue
+    'backports.zoneinfo==0.2.1;python_version<"3.9"',
+    "setuptools<66",
```

Comment:
https://stackoverflow.com/questions/71712258/error-could-not-build-wheels-for-backports-zoneinfo-which-is-required-to-insta 

https://github.com/pypa/setuptools/issues/3772#issuecomment-1667944589